Documentation/Maintenance/Release.md

```diff
@@ -626,14 +642,11 @@ generate, then:
 Historical note: Before ITK 3.8, the documentation used to be generated in a
 directory called `Documentation/Doxygen`.
 
-In `Public`, copy the documentation to `/projects/Insight/Doxygen`, and create
-a subdirectory `Insight34-doxygen/Documentation/Doxygen`.
-
-The final directory will look like
-`/projects/Insight/Doxygen/Insight34-doxygen/Documentation/Doxygen`, and at that
-level copy the `html` directory and the `InsightToolkit.tag` file.
-
-Finally, create symbolic link at `/projects/Insight/WWW/InsightDocuments/Web`.
+Unpackage the Doxygen tarball, then upload to the Kitware web server to serve
+at "https://itk.org/Doxygen$MAJOR_VERSION$MINOR_VERSION/html/". Contact
+Kitware Sys-admin to setup a new rsync target. After the uploade is complete,
```

Comment:
Sorry Matt, looks like you a spare "e" was introduced in the new commit: "uploade ".

Reply:
:eagle: :eye: